frontend/src/services/inviteService.ts

```diff
@@ -1,27 +1,33 @@
 import { comsAxios } from './interceptors';
 
-import type { InviteCreateOptions } from '@/types';
-
 const PATH = 'permission/invite';
 
 export default {
   /**
-   * @function getInvite
-   * Use an invite token
-   * @returns {Promise} An axios response
+   * @function createInvite
+   * Post an Invite, exclusive or on bucketId and objectId
+   * @param {string} bucketId
+   * @param {string} objectId
+   * @param {string} email to be used for access
+   * @param {string} expiration timestamp for token
+   * @returns {string} uuid token of the invite
    */
-  createInvite(params: InviteCreateOptions): Promise<string> {
-    return comsAxios().get(`${PATH}`, {
-      params: params
+  async createInvite(bucketId?: string, email?: string, expiresAt?: number, objectId?: string) {
```

Comment:
These functions do not need the async keyword on them as an await is not being used anywhere in the body.